src/test/rgw/rgw_multi/tests.py

```diff
@@ -1316,3 +1317,124 @@ def test_bucket_creation_time():
         for a, b in zip(z1, z2):
             eq(a.name, b.name)
             eq(a.creation_date, b.creation_date)
+
+
+#################
+# Workload tests
+#################
+
+@attr('workload_test')
+def test_basic_workload_beginner():
+    zonegroup = realm.master_zonegroup()
+    system_port = 8000
+    if len(zonegroup.zones) < 2:
+        raise SkipTest("test_basic_workload_beginner skipped. We're testing here with 2 or more zones in master zonegroup.")
+
+    zonegroup_conns = ZonegroupConns(zonegroup)
+    (zc1, zc2) = zonegroup_conns.rw_zones[0:2]
+
+    z1, z2 = (zc1.zone, zc2.zone)
+
+    # create a bucket on the first zone
+    bucket_name = gen_bucket_name()
+    log.info('create bucket zone=%s name=%s', z1.name, bucket_name)
+    bucket = zc1.conn.create_bucket(bucket_name)
+
+    system_access_key = z1.data['system_key']['access_key']
+
+    system_secret_key = z1.data['system_key']['secret_key']
+
+    system_host = "localhost:"+str(system_port)
+
+    subprocess.call(['warp', 'put', '--host', system_host, '--access-key', system_access_key, '--secret-key', system_secret_key, '--insecure', '--duration=10m', '--obj.size=10KiB', '--concurrent=6', '--noprefix', '--noclear', '--bucket', bucket_name])
+
+    zone_data_checkpoint(z2, z1)
+
```

Comment:
You still need to list the bucket on the primary and on the secondary and make sure that they contain the same objects.